rust/src/lib/ifaces/base.rs

```diff
@@ -77,12 +77,12 @@ pub struct BaseInterface {
     /// network interface activated.
     /// Serialize and deserialize to/from `wait-ip`.
     pub wait_ip: Option<WaitIp>,
-    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(skip_serializing_if = "Option::is_none", rename = "ipv4")]
```

Comment:
This rename has no effect. Why we need it?

Reply:
I think this is wrong  from my part at golang code the field is "IPv4" so my kebab-case convertions conert it to ipv-4, let me check if fixing that make this irrelevant.

Reply:
Fixed